samples/javascript_nodejs/51.teams-messaging-extensions-action/bots/teamsMessagingExtensionsActionBot.js

```diff
@@ -48,28 +137,37 @@ function shareMessageCommand(context, action) {
     let images = [];
     const includeImage = action.data.includeImage;
     if (includeImage === 'true') {
-        images = ['https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQtB3AwMUeNoq4gUBGe6Ocj8kyh3bXa9ZbV7u1fVKQoyKFHdkqU'];
+        images = [
+            'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQtB3AwMUeNoq4gUBGe6Ocj8kyh3bXa9ZbV7u1fVKQoyKFHdkqU',
+        ];
     }
-    const heroCard = CardFactory.heroCard(`${ userName } originally sent this message:`,
+    const heroCard = CardFactory.heroCard(
+        `${userName} originally sent this message:`,
         action.messagePayload.body.content,
-        images);
+        images
+    );
 
-    if (action.messagePayload.attachments && action.messagePayload.attachments.length > 0) {
+    if (
+        action.messagePayload.attachments &&
+        action.messagePayload.attachments.length > 0
+    ) {
```

Comment:
Should be able to simplify to:

```suggestion
    if (action.messagePayload.attachments?.length > 0) {
```